public/assets/main.css

```diff
@@ -339,3 +339,26 @@ div.game.card em {
   font-style: normal;
   text-decoration: underline;
 }
+
+#stock_market .splayed {
```

Comment:
i don't think this css is necessary here is opposed to a prop

Reply:
The class splayed is toggled. I’d have to toggle prop, don’t know if that works as easily.